.github/workflows/spell-checker.yml

```
@@ -12,9 +12,17 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
         continue-on-error: true
-        
\ No newline at end of file
+
+      - name: Vale Linter
+        uses: errata-ai/vale-action@v2.1.1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          files: all
+          reporter: github-pr-review
+          fail_on_error: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)